app/models/discussion.rb

```diff
@@ -8,10 +8,6 @@ class Discussion < ActiveRecord::Base
 
   default_scope -> { order('updated_at DESC') }
 
-  after_save :update_subscriptions
```

Comment:
Well, after_touch seems to work. I can leave it like that for now then, but I believe we'll eventually have to change to solutions like `CommentCreator` if this notifications system gets more complex.
But for now it seems to be enough